hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/jobhistory/TestHistoryViewerPrinter.java

```diff
@@ -160,6 +161,14 @@ public void testHumanPrinter() throws Exception {
         LINE_SEPARATOR, outStr);
   }
 
+  public static boolean compareStrings(String str1, String str2) {
+    String[] lines1 = str1.trim().split("\n");
+    String[] lines2 = str2.trim().split("\n");
+    Arrays.sort(lines1);
+    Arrays.sort(lines2);
+    return Arrays.equals(lines1, lines2);
```

Comment:
You don't use the output.
You maye want to call it something closer to assert and also do a final:
```
assertEquals(lines1, lines2)
```